eventClass/bundle.py

```diff
@@ -49,6 +56,49 @@ def __ne__(self, other):
         """
         return not self.__eq__(other)
 
+    def __str__(self):
```

Comment:
Once again this should be moved to a to_json method, the __str__ method should not be used to create a JSON representation.